guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,196 @@
 {
```

Comment:
<p><code><a href="https://developers.arcgis.com/python/api-reference/arcgis.raster.functions.gbl.html#" target="_blank">arcgis.raster.functios.gbl</a></code></p><p> Typo in functions </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>fixed</p>